templates/base/base.html

```diff
@@ -13,15 +13,15 @@
 {% include "base/navigation.html" %}
 
 <div class="govuk-width-container app-width-container">
-  <div class="govuk-phase-banner">
+  <!--<div class="govuk-phase-banner">
     <p class="govuk-phase-banner__content">
       <strong class="govuk-tag govuk-phase-banner__content__tag">Alpha
       </strong>
       <span class="govuk-phase-banner__text">
         This is a new service – your <a class="govuk-link" href="#">feedback</a> will help us to improve it.
       </span>
     </p>
```

Comment:
Instead of hiding the whole phase banner, I wonder if we should keep it there so it's clearly alpha, but hide the feedback link as it doesn't do anything?

We could say - "this service is under development" for the banner text?